jenkins/clone.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+NAME=$1
+PROJECT=$2
+BASE=".$NAME-base"
+
+# update our clone
+if [ ! -d .$NAME-base ]; then
```

Comment:
this script could do with some quotes around its $vars, so as to reduce breakage in the event of spaces in filenames